analyses/cell-type-wilms-tumor-06/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# pull base image
+FROM bioconductor/tidyverse:3.19
+
+
+# Change the default CRAN mirror
+RUN echo "options(repos = c(CRAN = 'https://mran.microsoft.com/snapshot/2022-02-01'), download.file.method = 'libcurl')" >> ${R_HOME}/etc/Rprofile.site
+
+
+# Set global R options
+RUN echo "options(repos = 'https://cloud.r-project.org')" > $(R --no-echo --no-save -e "cat(Sys.getenv('R_HOME'))")/etc/Rprofile.site
+ENV RETICULATE_MINICONDA_ENABLED=FALSE
+
+
+RUN R -e "devtools::install_github('enblacar/SCpubr')"
+RUN R --no-echo --no-restore --no-save -e "install.packages('remotes')"
+RUN R -e "remotes::install_github('satijalab/seurat', 'seurat5', quiet = TRUE)"  # this also install patchwork (and others)
+RUN R -e "remotes::install_github('satijalab/azimuth', quiet = TRUE)"            # this also install SingleCellExperiment, DT (and others)
```

Comment:
Thank you @jaclyn-taroni for your comment on the Dockerfile! FYI, I decided to rebuild an image based on the template you provided here : https://openscpca.readthedocs.io/en/latest/ensuring-repro/docker/docker-images/#r-based-images and try to take your comments into account! 

Also, I will follow your instructions to build the image: 
# move to the module directory
cd /path/to/your-module
# build the docker image
docker buildx build . -t openscpca/your-module:latest --platform linux/amd64

I am quite confident that this will solve the problem of building the docker image. :)